### ISSUE TITLE
statusHeat not good?

### ISSUE THREAD

**Issue by dorfl68** [jamesnw/statstat#13]
I am pretty sure the statusHeat flag is not working (I wanted to have some fun with it). When I look at the webpage source, I think that Control.Model.Property.statusHeat (which the code picks up) has no relation with the heat being on. (1) Control.Model.Property.statusCool is set to 1 at the same time, and (2) I can see that flag being 0 when the graphics shows that the heater is on. 

There is an interesting tag <div id="eqStatusHeating"> further down on the website. This seems to be followed by  class="" when the heater is on, and class="hidden" when not. Might be a better way to do this

**Comment by jamesnw** [comment]
I agree, statusHeat isn't working. Here's the code that determines that on the site-

```
 updateEquipmentStatus = function() {
        var isFanRunning = model.get(model.Property.isFanRunning);
        var eqOutputStatus = model.get(model.Property.equipOutputStatus);
        var isLost = model.get(model.Property.isLost);

        $(ui.eqStatusFanOn).css('display', !isLost && isFanRunning != undefined && isFanRunning && (eqOutputStatus == undefined || (eqOutputStatus != 1 && eqOutputStatus != 2)) ? "inline-block" : "none");
        $(ui.eqStatusHeating).css('display', !isLost && eqOutputStatus != undefined && (eqOutputStatus == 1) ? "inline-block" : "none");
        $(ui.eqStatusCooling).css('display', !isLost && eqOutputStatus != undefined && (eqOutputStatus == 2) ? "inline-block" : "none");
    };
```

isFanRunning and equipOutputStatus appear to be set in a different way than the other properties, and aren't present in the source code at all, so I'm not sure if we'll be able to scrape them directly. Your idea of scraping the output would make sense to me.
